### ISSUE TITLE
New Beta Features For Amazon Kindle Users please

### ISSUE THREAD

**Issue by Dleemuse** [AndBible/and-bible#991]
First I want to say that when I opened my AndBible on my phone this morning to do my weekly Sunday school homework, I was amazed by all the new beta features! I can't wait to re-tool some of my bookmark studies I have compiled over the years. The structure you have set up to do this in the new bookmark section, linking the notes is fantastic! I have been patiently waiting for this type of feature for years. It is very user friendly for those of us who are tech challenged. I am also going to love the StudyPad feature. There are so many more beta upgrades to explore I can't wait to get started.

I do want to please, please, entreat you not to forget about updating the new beta features for users of the AndBible App on Amazon. I know this is sometimes over-looked, because I have asked you to update it in the past, & you so graciously have done so. I use AndBible on my phone & my Kindle all the time. And for AndBible lovers who are iPhone users like my husband, so cannot get it on his phone (we know about pocket sword, it just doesn't measure up in any way to AndBible), the offering of the app on Amazon for the Kindle is something we hope can always be updated & we hope it never goes away!

Thank you so much to ALL who work so hard on making AndBible the BEST Bible app out there!

Debra

**Comment by tuomas2** [comment]
Thank you so much for the feedback :-) If you want to update android devices without Play store (i.e. Kindle), you can always download beta apk files from github releases section (see those that are marked with "-beta"): https://github.com/AndBible/and-bible/releases (latest being at the moment  3.4.514-beta). 

I will upload only stable versions to Amazon store. Stable version (with translations and bugfixes) will be released in few months, Lord willing, if we get translations in time, and there are no serious bugs/issues.

**Comment by Dleemuse** [comment]
Thank you Tuomas. 
I am not sure what I am doing wrong. I followed the link in your email, clicked on "Release 3.4.514-beta," clicked "andbibke-beta-514.apk," clicked "install." It tells me the app is not installed, but when I open my And Bible App select menu in upper left then go to "download document from zip," I can see that it downloaded the beta, it shows up as "and-bibke-beta-514.zip" in my download folder in internal storage at 6.57MB. When I click on it, it says "file does not contain a valid sword module." 
I know I am missing something sorry! I would appreciate any help you can offer! 
Thanks, 
Debra
Sent from Yahoo Mail on Android 
 
  On Sat, Apr 24, 2021 at 11:34 AM, Tuomas ***@***.***> wrote:   


Thank you so much for the feedback :-) If you want to update android devices without Play store, you can always download beta apk files from github releases section (see those that are marked with "-beta"): https://github.com/AndBible/and-bible/releases (latest being at the moment 3.4.514-beta).

I will upload only stable versions to Amazon store. Stable version (with translations and bugfixes) will be released in few months, Lord willing, if we get translations in time, and there are no serious bugs/issues.

**Comment by timbze** [comment]
Hi @Dleemuse, make sure it is the .apk file that you download. Then install it by clicking the .apk file. If it does not work you may need to enable installing apps by .apk in your Kindle system settings. If there's an error message when you try to install the .apk file, maybe that will give a hint on what needs to be done.

"Download documents from zip" has nothing to do with the app update.

**Comment by timbze** [comment]
TL;DR beta won't be released on Amazon app store, but users are able to download beta apk releases from [release page](https://github.com/AndBible/and-bible/releases).
